ci-operator/jobs/openshift/cluster-openshift-apiserver-operator/openshift-cluster-openshift-apiserver-operator-release-4.4-presubmits.yaml

```diff
@@ -495,7 +495,6 @@ presubmits:
       ci-operator.openshift.io/prowgen-controlled: "true"
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-cluster-openshift-apiserver-operator-release-4.4-verify-deps
-    optional: true
```

Comment:
this will fast forwarded from master, so it will be dropping the check which is correct